test/controllers/notifications_controller_test.rb

```diff
@@ -12,8 +12,8 @@ class NotificationsControllerTest < ActionController::TestCase
   end
 
   test "should assign unapproved sign slides" do
-    users(:two).signs << signs(:one)
-    signs(:one).slides << slides(:two)
+    users(:two).signs << signs(:default)
```

Comment:
Ross and James have more detailed information that the other users don't have and removing them breaks a lot of tests, they won't be simple to fix